builtin/credential/cert/path_certs.go

```diff
@@ -133,7 +133,7 @@ func (b *backend) pathCertRead(
 		Data: map[string]interface{}{
 			"certificate":  cert.Certificate,
 			"display_name": cert.DisplayName,
-			"policies":     strings.Join(cert.Policies, ","),
+			"policies":     cert.Policies,
```

Comment:
Any concern with the behavior change of the return here?  I like this better but this does break compatibility.